ansible/configs/ocp-workshop/README.adoc

```diff
@@ -34,7 +34,7 @@ ansible-playbook ansible/main.yml \
   -e "cloud_provider=${CLOUDPROVIDER}" \
   -e "aws_region=${REGION}" \
   -e "HostedZoneId=${HOSTZONEID}" \
-  -e "key_name=${KEYNAME}" \
+  -e "ssh_provision_key_name=${KEYNAME}" \
```

Comment:
we should remove entirely that i think.
The name has a default, if you provide it, you're in the case "i already have a key, i provide it".